### DISCUSSION TITLE
Does Add BatchRegistrants for Meeting and Webinar implimented?

### DISCUSSION THREAD

**Discussion by maaaruf** [Jericho/ZoomNet#273] [Q&A] [answered]
I'm looking for these two APIs, [MeetingBatchRegistants](https://marketplace.zoom.us/docs/api-reference/zoom-api/methods/#operation/addBatchRegistrants) and [WebinarBatchRegistants](https://marketplace.zoom.us/docs/api-reference/zoom-api/methods/#operation/addBatchWebinarRegistrants). Have these APIs already been implemented? After looking at the interfaces, I couldn't find them.

**Comment by Jericho** [comment, marked as answer]
No it's not implemented. In fact, I created an issue quite a while ago to indicate it's not implemented and that it should be implemented at some point: #184

**Comment by maaaruf** [comment]
May I make a pull request for it? How long it will take to be available at nuget.org?

**Reply by Jericho** [reply]
For sure you can. I would be happy to take your PR.
